.github/issue_template.md

```diff
@@ -0,0 +1,21 @@
+_**NOTE:** this issue system is intended for reporting bugs and tracking progress in software development. Although this software is licensed with an open-source license, any issue opened here may not be responded to in a timely manner. [Conveyal](https://www.conveyal.com) is unable to provide technical support for custom deployments of this software unless your company has a support contract with us. Please remove this note when creating the issue._
+
+## Observed behavior
+
+Please explain what is being observed within the application here.
+
+## Expected behavior
+
+Please explain what should happen instead.
+
+## Steps to reproduce the problem
+
+Please be as specific as possible.
+
+## Any special notes on configuration used
+
+Please describe any applicable config files that were used
+
+## Version of datatools-server (exact commit hash or branch name)
```

Comment:
Perhaps we should optionally have issue creators include the datatools-ui commit hash/branch name if applicable (or maybe always).